README.md

```diff
@@ -99,6 +99,41 @@ from the Debug > Theme menu. There is not yet a mechanism for including
 custom themes.
 
 
+## CLI
+
+Xi-Mac includes a CLI for opening files directly from the command line.
+
+### Installing
+
+1. Build a release build of XiCli in Xcode  
```

Comment:
Can we figure out how to build this as part of the default scheme?

Reply:
We could create a 'release' scheme that builds both as a release. That would allow for ease of use without increasing debug build time for either.

Reply:
That's true; build time shouldn't be a major factor, since Xcode shouldn't rebuild `XiCli` unless it has changed?

Reply:
It would also allow it to be a release build without having to change settings every time

Reply:
A post-action could install XiCli to /usr/local/bin/xi in the release scheme. Do you think that's a good idea?

Reply:
That would work for us, but we want to think about when we're distributing a package to users; that's the case where we want a precompiled binary in the app bundle, and we can install it when requested.

I'm not sure I understand your second last comment, about changing settings? It should be a debug or a release build based on what's selected at build time.

Reply:
Are there any advantages of putting the CLI inside the app package? It seems like it would just add to the application while never being used for anything more than a copy. We could do what VSCode does by asking users to run a command to install the CLI after installing the main app.

Reply:
The advantage is just that we _can_ install it, at some point, if the user asks us to. This will be what VSCode is doing, too; when you install the CLI,  it creates a bash script in `usr/local/bin` that loads a file, `CLI.js`, that is included in VSCode's app bundle. This is similar to what we currently do, if you install the simple CLI from the the 'XiEditor' menu.

Does that make sense? There isn't really a viable alternative to including the CLI in the app package (that I know of), if we want to distribute a pre-compiled binary, which we will want eventually.

Reply:
Ohhh, I understand now. Sorry, I just wasn't realizing the install would be through the app instead of the command line. I'll work on getting that button to install the new CLI.

Reply:
Got it installing from XiEditor!